kolibri/core/logger/models.py

```diff
@@ -237,8 +243,8 @@ class Meta:
 
 class AttemptLog(BaseAttemptLog):
     """
-    This model provides a summary of a user's engagement within a particular interaction with an
-    item/question in an assessment
+    This model provides a summary of a user's interactions with a question in an exercise.
```

Comment:
This is inaccurate, as soon we will have assessments that are not exercises.

Reply:
That's what I was trying to clarify earlier when I renamed the classes. My understanding was that `AttemptLog` was specifically for exercises, and `BaseAttemptLog` is the base class?

Reply:
`AttemptLog` is for any assessment attempt taken within the context of a `ContentNode`, `BaseAttemptLog` is the base class for `AttemptLog` and `ExamAttemptLog`. `ExamAttemptLog` is specifically for assessment attempts taken within the context of Quizzes (nee exams).